### ISSUE TITLE
Create ios and Android app

### ISSUE THREAD

**Issue by pke** [milgner/keedrop#22]
For the executives who can't navigate a website but prefer an app for everything ;)

That's more an exercise in app creation using frameworks like ionic/react or react native.

Its also a nice playground for CI/CD tooling offered by https://fastlane.tools/

**Comment by milgner** [comment]
What about just providing it as a PWA?

**Comment by pke** [comment]
That's already in the works :)

**Comment by milgner** [comment]
@pke I was just thinking of using this is a nice pet project to try out Flutter... :innocent:

**Comment by pke** [comment]
maybe. I have to check flutter. Its rendering on a canvas, iirc. Not sure how the accessibility is there. The TechLead on youtube had something to say about that.
We can try and fall back to React Native ;)

**Comment by milgner** [comment]
Yes, React Native would have been another option - but I feel like I already have a good impression on that. Would like to get some insight into Flutter, too :)

**Comment by pke** [comment]
Well here is some insight that frightens me
https://github.com/flutter/flutter/issues/51752

**Comment by milgner** [comment]
That is frightening! We should have a GPT-3 algorithm build the app then we don't need to write any boilerplate 😝

Or let's stick with React Native...
